scripts/apps/vocabularies/controllers/VocabularyEditController.ts

```diff
@@ -48,7 +48,7 @@ export function VocabularyEditController(
     $scope.idRegex = idRegex;
     $scope.selectionTypes = VOCABULARY_SELECTION_TYPES;
 
-    if ($scope.matchFieldTypeToTab?.('related-content-fields', $scope.vocabulary.field_type)) {
+    if ($scope.matchFieldTypeToTab.('related-content-fields', $scope.vocabulary.field_type)) {
```

Comment:
You forgot to remove [the dot](https://user-images.githubusercontent.com/2076953/70918546-7eb3a980-201f-11ea-9008-54156bc0c407.png) from the function call.

You should probably also pass the [same thing](https://user-images.githubusercontent.com/2076953/70918803-fa155b00-201f-11ea-8e7e-3f27104376da.png) from the tests

Reply:
Yeah I already fixed the dot. And you're right about the noop fn, I did mean to have the same thing in both